source/scaling.rst

```diff
@@ -0,0 +1,22 @@
+.. _operator-scale:
+
+Scale Percona Distribution for PostgreSQL on Kubernetes and OpenShift
+=====================================================================
+
+One of the great advantages brought by Kubernetes and the OpenShift
+platform is the ease of an application scaling. Scaling an application
+results in adding or removing the Pods and scheduling them to available 
+Kubernetes nodes.
+
+Size of the cluster is controlled by a :ref:`pgReplicas.<replica-name>.size key<pgreplicas-size>` in the :ref:`operator.custom-resource-options` configuration.  That’s why scaling the cluster needs nothing more but changing
+this option and applying the updated configuration file. This may be done in a
+specifically saved config, or on the fly, using the following command:
+
+.. code:: bash
+
+   $ kubectl scale --replicas=5 pgo/cluster1
+
+
+In this example we have changed the size of the PostgreSQL Cluster to ``5``
+replica instances. 
```

Comment:
I would emphasize here, that we change only the number of Replicas. So the size of the cluster here should be 5 Replicas + 1 Primary.